### PR TITLE
Replace service

### PR DESCRIPTION
**Pull request by jenmerritt**
### Description

- Replaces the service for this sample to match other platforms that have replaced the flaky ss6 service.
- Relevant updates to Readme and screenshot

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

**Comment by jenmerritt** [comment]
@Luiy0 would you be able to review?

**Review by Luiy0 — APPROVED** [review]
Looks good @jenmerritt

**Comment by jenmerritt** [comment]
@Rachael-E Please could I get a second review?

File: ogc/wmts-layer/README.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ Pan and zoom to explore the WMTS layer, which is displayed automatically.
 ## How it works
  
 1. Create a `WmtsService` using the URL of the WMTS Service.
-1. After loading the WMTS service, get the list of `WmtsLayerInfos` from the service info: `service.getServiceInfo().getLayerInfos()`
-1. Use one of the layer infos to create a new `WmtsLayer(layerInfos.get(0))`
-1. Set it as the map's basemap with `map.setBasemap(new Basemap(wmtsLayer))`.
+2. After loading the WMTS service, get the list of `WmtsLayerInfos` from the service info: `service.getServiceInfo().getLayerInfos()`
+3. Use one of the layer infos to create a new `WmtsLayer(layerInfo)`
```
Comment (Rachael-E) [inline ogc/wmts-layer/README.md]: just to make this super clear what parameter is being passed:

```suggestion
3. Create a new `WmtsLayer` from a `WmtsLayerInfo`
```

File: ogc/wmts-layer/src/main/java/com/esri/samples/wmts_layer/WmtsLayerSample.java [hunk 2]
```diff
@@ -58,15 +59,17 @@ public void start(Stage stage) {
...
       // create a WMTS service from a URL
-      String serviceURL = "https://sampleserver6.arcgisonline.com/arcgis/rest/services/WorldTimeZones/MapServer/WMTS";
+      String serviceURL = "https://gibs.earthdata.nasa.gov/wmts/epsg4326/best";
       wmtsService = new WmtsService(serviceURL);
       wmtsService.addDoneLoadingListener(() -> {
         if (wmtsService.getLoadStatus() == LoadStatus.LOADED) {
           WmtsServiceInfo wmtsServiceInfo = wmtsService.getServiceInfo();
-          // get the first layer's ID
-          List<WmtsLayerInfo> layerInfos = wmtsServiceInfo.getLayerInfos();
+          // Obtain the read only list of WMTS layer info objects, and select the one with the desired Id value.
```
Comment (Rachael-E) [inline ogc/wmts-layer/src/main/java/com/esri/samples/wmts_layer/WmtsLayerSample.java]:
```suggestion
          // obtain the read only list of WMTS layer info objects, and select the one with the desired Id value.
```

File: ogc/wmts-layer/README.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ Pan and zoom to explore the WMTS layer, which is displayed automatically.
 
 ## About the data
 
-The map visualizes world time zones.
+We acknowledge the use of imagery provided by services from the [Global Imagery Browse Services (GIBS)](https://wiki.earthdata.nasa.gov/display/GIBS/), operated by the [NASA/GSFC/Earth Science Data and Information System (ESDIS)](https://earthdata.nasa.gov/) with funding provided by NASA/HQ.
```
Comment (Rachael-E) [inline ogc/wmts-layer/README.md]: We could add a bit more information here about the data (I checked this looking at the capabilities xml: https://gibs.earthdata.nasa.gov/wmts/epsg4326/best/1.0.0/WMTSCapabilities.xml):

```suggestion
We acknowledge the use of imagery provided by services from the [Global Imagery Browse Services (GIBS)](https://wiki.earthdata.nasa.gov/display/GIBS/), operated by the [NASA/GSFC/Earth Science Data and Information System (ESDIS)](https://earthdata.nasa.gov/) with funding provided by NASA/HQ. This sample shows the Digital Elevation Model (Color Index, SRTM) layer from the WMTS service provided by GIBS, using the "SRTM_Color_Index" identifier. 
```

**Review by Rachael-E — APPROVED** [review]
looks good @jenmerritt , couple of suggestions to consider

**Pull request merged**